src/node.cc

```diff
@@ -3279,6 +3280,7 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_forceRepl", True(env->isolate()));
   }
 
+  // -r,--require
```

Comment:
Actually can you add a space here after the comma to be consistent with the other existing comments?